lib/awful/titlebar.lua

```diff
@@ -551,6 +578,7 @@ end
 --   "right", "top", "bottom". Default is "top".
 function titlebar.toggle(c, position)
     position = position or "top"
+    if load_titlebars(c, true, position) then return end
```

Comment:
Why this new early return? This should break any keybindings that toggle titlebar visibility. I can also come up with an example where the new early return in `titlebar.show` breaks stuff.

Reply:
Because if `load_titlebar` returns true, then it is already shown

Reply:
Okay, and who hides it then?

Reply:
toggle?

If load_titlebar load something, then toggle will hide it. But if the user call toggle and there is no titlebar, then he/she want to show it?

Otherwise I don't understand the question

Reply:
Okay, so: .... oh wait, I got the return value of `load_titlebars` wrong. Sorry!